requirements.pip

```diff
@@ -1,14 +1,16 @@
 --extra-index-url https://staticfiles.krxd.net/foss/pypi/
 # direct dependencies
 sh==1.11
-virtualenv-tools==1.0
+# note that this virtualenv-tools is our fork, not the public one, the source is
+# here: https://github.com/krux/virtualenv-tools
+virtualenv-tools==1.0.1
```

Comment:
Is 1.0.1 our version? Will there be potential for version number clashes between our fork and official upstream?